test/acceptance/dashboard.virtualHost.spec.js

```diff
@@ -0,0 +1,92 @@
+'use strict';
+
+const _ = require('lodash');
+const parseUrl = require('url').parse;
+const lib = require('../../lib');
+const app = require('../../apps').external;
+const fabrik = lib.fabrik;
+const virtualHostStore = lib.iaas.virtualHostStore;
+const config = lib.config;
+
+describe('dashboard', function () {
+  describe('virtualHost', function () {
+
+    const service_id = '19f17a7a-5247-4ee2-94b5-03eac6756388';
+    const plan_id = 'd035f948-5d3a-43d7-9aec-954e134c3e9d';
+    const plan_guid = 'f6280923-b144-4f02-adf7-76a7b5ef3a4a';
+    const index = mocks.director.networkSegmentIndex;
+    const instance_id = mocks.director.uuidByIndex(index);
+    const parent_instance_id = 'b4719e7c-e8d3-4f7f-c515-769ad1c3ebfa';
+    const deployment_name = mocks.director.deploymentNameByIndex(index);
+    const filename = `virtual_hosts/${instance_id}/${instance_id}.json`;
+    const container = virtualHostStore.containerName;
+    const pathname = `/${container}/${filename}`;
+    const data = {
+      instance_guid: instance_id,
+      deployment_name: deployment_name
+    };
+
+    before(function () {
+      _.unset(fabrik.VirtualHostManager, plan_id);
+      virtualHostStore.cloudProvider = new lib.iaas.CloudProviderClient(config.virtual_host.provider);
+      mocks.cloudProvider.auth();
+      mocks.cloudProvider.getContainer(container);
+      return mocks.setup([
+        virtualHostStore.cloudProvider.getContainer()
+      ]);
+    });
+
+    afterEach(function () {
+      mocks.reset();
+    });
+
+    describe('/manage/instances/:service_id/:plan_id/:instance_id', function () {
+      this.slow(1500);
```

Comment:
why is this requried? We generally dont have such overrides. Any reason these tests might take longer?

Reply:
Not fully sure of times taken by all the mocks, but it is also used at docker dashboard unit tests https://github.com/cloudfoundry-incubator/service-fabrik-broker/blob/master/test/acceptance/dashboard.docker.spec.js#L35,
also the CI build failed because of default timeout.

Reply:
quite interesting. It would be worth investigating some other time. For now then this is ok.